pandas/tests/reshape/test_concat.py

```diff
@@ -610,11 +610,11 @@ def test_concat_categorical_3elem_coercion(self):
         s2 = pd.Series([2, 1, 2], dtype="category")
         s3 = pd.Series([1, 2, 1, 2, np.nan])
 
-        exp = pd.Series([1, 2, np.nan, 2, 1, 2, 1, 2, 1, 2, np.nan], dtype="object")
+        exp = pd.Series([1, 2, np.nan, 2, 1, 2, 1, 2, 1, 2, np.nan], dtype="float")
```

Comment:
this would be an api change can you add a release note

Reply:
Yes, but can you check https://github.com/pandas-dev/pandas/pull/33607#issuecomment-615168747 and indicate if we are fine with that change?